elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydrator.java

```diff
@@ -0,0 +1,93 @@
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Template;
+import com.github.jknack.handlebars.helper.ConditionalHelpers;
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.TemplateLoader;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for handlebars hydration.
+ */
+public class HandlebarsHydrator {
+
+    public static final EscapingStrategy MY_ESCAPING_STRATEGY = new Hbs(new String[][]{
+        {"<", "&lt;" },
+        {">", "&gt;" },
+        {"\"", "&quot;" },
+        {"`", "&#x60;" },
+        {"&", "&amp;" }
+    });
+
+    /**
+     * Method to hydrate the Table template.
+     * @param table ElideTable object
+     * @return map with key as table java class name and value as table java class definition
+     * @throws IOException IOException
+     */
+    public Map<String, String> hydrateTableTemplate(ElideTable table) throws IOException {
+
+        Map<String, String> tableClasses = new HashMap<>();
+
+        TemplateLoader loader = new ClassPathTemplateLoader("/templates");
+        Handlebars handlebars = new Handlebars(loader).with(MY_ESCAPING_STRATEGY);
+        HandlebarsHelper helper = new HandlebarsHelper();
+        handlebars.registerHelpers(ConditionalHelpers.class);
+        handlebars.registerHelpers(helper);
+        Template template = handlebars.compile("table");
+
+        for (Table t : table.getTables()) {
+            tableClasses.put(helper.capitalizeFirstLetter(t.getName()), template.apply(t));
+        }
+
+        return tableClasses;
+    }
+
+    /**
+     * Method to replace variables in hjson config.
+     * @param config hjson config string
+     * @param replacements Map of variable key value pairs
+     * @return hjson config string with variables replaced
+     * @throws IOException IOException
+     */
+    public String hydrateConfigTemplate(String config, Map<String, Object> replacements) throws IOException {
+
+        Context context = Context.newBuilder(replacements).build();
+        Handlebars handlebars = new Handlebars();
+        Template template = handlebars.compileInline(config);
+
+        return template.apply(context);
+    }
+
+    /**
+     * Method to hydrate the Security template.
+     * @param security ElideSecurity Object
+     * @return security java class string
+     * @throws IOException IOException
+     */
+    public String hydrateSecurityTemplate(ElideSecurity security) throws IOException {
+
+        TemplateLoader loader = new ClassPathTemplateLoader("/templates");
+        Handlebars handlebars = new Handlebars(loader).with(MY_ESCAPING_STRATEGY);
+        handlebars.registerHelpers(ConditionalHelpers.class);
+        handlebars.registerHelpers(new HandlebarsHelper());
+        Template template = handlebars.compile("security");
+
+        return template.apply(security);
```

Comment:
As mentioned in the other comment, this should return multiple classes by looping the individual entries in roles. So this class should also return a map like hydrateTableTemplate.